server/grpc_service.go

```diff
@@ -1034,23 +1033,12 @@ func (s *Server) GetDCLocations(ctx context.Context, request *pdpb.GetDCLocation
 	if err := s.validateInternalRequest(request.GetHeader(), false); err != nil {
 		return nil, err
 	}
-	if !s.member.IsLeader() {
+	if !s.member.IsStillLeader() {
```

Comment:
why rename this?

Reply:
Theoretically, `IsStillLeader` is more accurate than `IsLeader` to check the leader role.